calculator.js

```diff
@@ -5,8 +5,8 @@
 
```

Comment:
The file is a bit big, maybe refactor some of the functionallity, move it to a new file. Find a smart way to group the functionallity

Reply:
dont focus on this now, lets make this baby work first! then i can give it a shot